hail/python/hail/methods/impex.py

```diff
@@ -3161,13 +3171,15 @@ def add_reference_allele(mt):
 
         var_ht = hl.import_avro(var_group)
         var_ht = var_ht.transmute(locus=translate_locus(var_ht.location),
-                          local_alleles=var_ht.alt.split(','),
-                          LGT=hl.parse_call(var_ht.GT),
-                          GQ=hl.int32(var_ht.GQ),
-                          RGQ=hl.int32(var_ht.RGQ))
+                                  local_alleles=var_ht.alt.split(','),
+                                  LGT=hl.parse_call(var_ht.GT),
+                                  LAD=var_ht.AD.split(',').map(lambda x: hl.int32(x)),
+                                  GQ=hl.int32(var_ht.GQ),
+                                  RGQ=hl.int32(var_ht.RGQ))
```

Comment:
AS_QUALapprox and QUALapprox would be welcome additions, I will add them to the spec/ query